rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -2,6 +2,45 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[december-12-2023]]
+== December 12, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* RPA Manager now implements Anypoint Audit Logging, which enables you to see a queryable history of actions performed with RPA objects.
+* The *Usage Summary* is now grouped the process category.
+* The *Usage Summary* now shows the Cumulated Bot Run Time and Average Bot Run Time additionally in _hh:mm:ss_ format.
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Upcoming Process Changes* view no longer shows an error when selecting the table header *Last edited*. | W-14293208
+| *Process Run Results* no longer show *Endtime* and *Duration* if the respective process is in status _running_. | W-14311008
+| You can now download RPA Bot logs in the *Bot Management* when the RPA Bot is in status _out of service time_. | W-14378109
+| You can now use the same name for global variables and credentials in different organizations. Names are unique within the same organization. | W-14378876
+| The close button of the videos player in the home page no longer dissappears after resizing the browser's window. | W-14387182
+| You can now create a global variable of type integer with a negative value. | W-13460628
+| The dashboard widget *Business Process Performance* now aggregates successful and failed process executions. | W-14438065
```

Comment:
```suggestion
| The dashboard widget *Business Process Performance* now aggregates successful and failed process executions. | W-14438065
```
is *failed* correct?

Reply:
Yes, to make it more clear maybe I can reword to: _now aggregates **both** successful and failed process executions._

Reply:
**Both** works fine!